src/features/all-drops/components/AllDrops.tsx

```diff
@@ -193,16 +193,38 @@ export default function AllDrops() {
     [pagination],
   );
 
-  useEffect(() => {
+  const handleMultipleEffects = async () => {
     if (!accountId) return;
+
     // page query param should be indexed from 1
     const pageQuery = searchParams.get('page');
     const currentPageIndex = pageQuery !== null ? parseInt(pageQuery) - 1 : 0;
     setPagination((pagination) => ({ ...pagination, pageIndex: currentPageIndex }));
     handleGetDropsSize();
     handleGetDrops({ start: currentPageIndex * pagination.pageSize });
-    handleFinishNFTDrop();
-  }, [accountId, searchParams]);
+
+    // first mount, check for NFT attempt
+    if (refreshTrigger === '') {
```

Comment:
no hard page reloads, if we rename it refreshDrops then it only refreshes drops

right now it can be used inside any pages, could be used to refresh keys for example

call it a utility for now